### PR TITLE
Added LongClickActions

### PR DESCRIPTION
**Pull request by g00m**
I added the possibility to interact with a view by using a long press viewactions.
Tests were also added to make sure the long press action is working correctly.

### CODE REVIEW

File: library/src/main/java/com/schibsted/spain/barista/BaristaClickActions.java [hunk 1]
```diff
@@ -25,64 +28,82 @@
 
 public class BaristaClickActions {
 
-  private static final ResourceTypeChecker RESOURCE_TYPE_CHECKER = new ResourceTypeChecker();
+    private static final ResourceTypeChecker RESOURCE_TYPE_CHECKER = new ResourceTypeChecker();
```
Comment (Sloy) [inline library/src/main/java/com/schibsted/spain/barista/BaristaClickActions.java]: I think you accidentally changed the indentation of this file. In this project we use a 2-spaces indentation. Please, could you revert it?

File: sample/src/androidTest/java/com/schibsted/spain/barista/sample/LongClickTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+  @Test
+  public void checkLongClick_byText() {
+    longClick("Next");
+    assertDisplayed("I was long pressed");
+  }
+
+  @Test
+  public void checkClick_byStringResource() {
```
Comment (Sloy) [inline sample/src/androidTest/java/com/schibsted/spain/barista/sample/LongClickTest.java]: I think you renamed the two first tests with "LongClick" but forgot to update the rest. Could you change them so they're not duplicate with the normal click tests?

**Comment by g00m** [comment]
Thank you for your response. Im going to fix the stuff you mentioned. Can I commit the changes to my current branch or should i create a "clean" new one? :)

**Comment by Sloy** [comment]
Yes @g00m, you can totally keep working on the same branch. Thanks!

**Comment by g00m** [comment]
So I updated the files as you requested. Hopefully everything is now how it should be. If there is something else just let me know :)

**Comment by Sloy** [comment]
There were still come conflicts due to our code style configurations. I just updated the code, I think it's easier since I already have my IDE configured with it ;)

Let's see if it works now!

**Comment by Sloy** [comment]
🎉

**Comment by Sloy** [comment]
Forgot to say @g00m but your change is already available in version 1.8.0 on JCenter.
Thanks a lot again for the contribution, and happy coding ;)

**Comment by g00m** [comment]
Nice thanks a lot. Until next time ;-)

**Pull request merged**